commit 88a74443eb0a6f395d8210deb967db5afac3314b (refs/changes/48/4248/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-06-18T15:38:03+00:00 (1 year, 4 months ago)
    
    OS-7032 mismatch between sysinfo (json) and sysinfo -p
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Rui Loura <rui.loura@joyent.com>

diff --git a/src/sysinfo b/src/sysinfo
index c28215f2..2a889c77 100644
--- a/src/sysinfo
+++ b/src/sysinfo
@@ -937,7 +937,7 @@ END
         ipv4_var="Virtual_Network_Interface_${iface}_IPv4_Address"
         link_status_var="Virtual_Network_Interface_${iface}_Link_Status"
         vlan_var="Virtual_Network_Interface_${iface}_VLAN"
-        parent_var="Virtual_Network_Interface_${iface}_Host_Interface"
+        host_var="Virtual_Network_Interface_${iface}_Host_Interface"
 
         eval "mac=\${${mac_var}}"
         eval "ipv4=\${${ipv4_var}}"
